packages/zk-eddsa-frog-pcd/src/ZKEdDSAFrogPCD.ts

```diff
@@ -0,0 +1,402 @@
+import {
+  EdDSAFrogPCD,
+  EdDSAFrogPCDPackage,
+  EdDSAPCDTypeName,
+  IFrogData,
+  frogDataToBigInts
+} from "@pcd/eddsa-frog-pcd";
+import type { EdDSAPublicKey } from "@pcd/eddsa-pcd";
+import {
+  ArgumentTypeName,
+  BigIntArgument,
+  DisplayOptions,
+  PCD,
+  PCDArgument,
+  PCDPackage,
+  ProveDisplayOptions,
+  SerializedPCD
+} from "@pcd/pcd-types";
+import {
+  SemaphoreIdentityPCD,
+  SemaphoreIdentityPCDPackage
+} from "@pcd/semaphore-identity-pcd";
+import { STATIC_SIGNATURE_PCD_NULLIFIER } from "@pcd/semaphore-signature-pcd";
+import {
+  fromHexString,
+  generateSnarkMessageHash,
+  hexToBigInt,
+  requireDefinedParameter
+} from "@pcd/util";
+import {
+  Groth16Proof,
+  prove as groth16Prove,
+  verify as groth16Verify
+} from "@zk-kit/groth16";
+import { Eddsa, buildEddsa } from "circomlibjs";
+import JSONBig from "json-bigint";
+import { v4 as uuid } from "uuid";
+import vkey from "../artifacts/circuit.json";
+import { ZKEdDSAFrogCardBody } from "./CardBody";
+
+export const STATIC_ZK_EDDSA_FROG_PCD_NULLIFIER = generateSnarkMessageHash(
+  "hard-coded-zk-eddsa-frog-pcd-nullifier"
+);
```

Comment:
add a comment explaining that this is an optional default, not one that must be used for all proofs.